_posts/en/newsletters/2021-04-21-newsletter.md

```diff
@@ -0,0 +1,247 @@
+      alternative to fee bumping.  For LN, sometimes waiting isn't an
+      option---a fee bump will need to be submitted within a matter of
+      hours or funds could be lost.
+
+    - *Timelocked outputs:* for most regular onchain payments, a user
+      who wants to CPFP bump can pay for a fee bump using the funds
+      stored in their output from the transaction they want to bump.  In
+      the case of LN, those funds aren't available until the channel
+      close has been fully settled onchain.  That means the user needs
+      to use a separate UTXO to pay the fees.
+
+    To address tho above two concerns, LND is requiring users of anchor
+    outputs to retain at least one confirmed UTXO of reasonable value in
+    their wallet any time a channel is open.  That ensures they can CPFP
+    fee bump when necessary, but it has certain consequences, such as
+    preventing spending the last of your onchain funds (even to open a
+    new channel) while you still have at least one channel open.
+
+    Osuntokun's request is for wallets or services built on LND to let
+    the development team know if any of the above concerns, or any other
+    concerns related to anchor outputs, will cause serious problems.
+    Although the question is specific to LND, the answers may have
+    implications for all LN nodes.
+
+- **Sapio public launch:** Jeremy Rubin [posted][rubin sapio] to the
+  Bitcoin-Dev mailing list an announcement that he has made available
+  the Sapio smart contract development toolkit, a Rust-based library
+  and associated tooling that can be used to create smart contracts
+  expressible using Bitcoin Script.  The language was originally
+  designed to make use of Rubin's proposed
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] opcode
+  (`OP_CTV`) but it can simulate the existence of that opcode, and of
+  other potential features for Bitcoin such as taproot, using a trusted
+  signing oracle.  In addition to the Sapio library, the release also
+  contains extensive documentation and an experimental frontend.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Specter v1.3.0 released:**
+  [Specter v1.3.0][] includes additional RBF transaction support, Bitcoin Core setup
+  from within the application, HWI 2.0.0 support, and an option to use [mempool.space][news132
+  mempool.space] as a [block explorer][topic block explorers] and for fee estimation.
+
+- **Specter-DIY v1.5.0:**
+  Hardware wallet firmware Specter-DIY [released][specter-diy github] v1.5.0 which adds custom SIGHASH
+  flag support and full [descriptor][topic descriptors] support including [miniscript][topic miniscript].
+
+- **ION v1.0.0 launches on Bitcoin mainnet:**
+  Decentralized identity network [ION][ion github] is a layer 2 protocol for
+  [decentralized identifiers][did w3c] that anchors records into a Bitcoin transaction.
```

Comment:
I don't think this fits in the "Changes to services and client software" section.

Reply:
Seems to me it's a "service" but one that's "built on top of bitcoin" rather than a "bitcoin service" ? I was going to say maybe treat it like Open Timestamps, but that seems never to have been mentioned in optech... Can't see much in the way of mentions of the RSK merge-mined chain or RGB coloured coins which also seem comparable, and I think all the other similar ideas are still R&D phase rather than being active services/clients. So yeah, maybe it should be under "News" ?

Reply:
This has been removed. Thanks for the feedback guys.